lldb/test/API/tools/lldb-server/TestAppleSimulatorOSType.py

```diff
@@ -39,7 +39,7 @@ def check_simulator_ostype(self, sdk, platform_name, arch=platform.machine()):
             for device in devices:
                 if "availability" in device and device["availability"] != "(available)":
                     continue
-                if "isAvailable" in device and device["isAvailable"] != True:
+                if "isAvailable" in device and device["isAvailable"] is not True:
```

Comment:
Can this be simplified to `if "isAvailable" in device and not device["isAvailable"]:`?

Reply:
fixed